package.json

```diff
@@ -32,6 +32,7 @@
     "setup:build": "npm run setup --build",
     "setup": "npm run clean:all && npm install ./packages/strapi-lint --save-dev && npm install && node ./scripts/setup.js && npm run clean",
     "lint": "node ./scripts/lint.js",
+    "translations": "node ./scripts/translations.js && git add .",
```

Comment:
First change I see, and this imposes a bad workflow by taking away the option to be able to control the commits.

This is a really bad assumption.

Reply:
I'm not sure to understand what you mean.

Reply:
🤔 I think the image is clear; obviously is under this branch.

1. I made changes on 3 files
2. I try to commit **just one file**
3. Was not possible because the hook do `git add .`. At the end, the three files were commited

Partial commits is good feature used by a medium git user. This changes just remove whatever control of what the user wants to commit...

What I suggest?; don't put it as a pre-commit hook. You can put it as a part of the CI flow if you want.

Reply:
We can apply:

```suggestion
    "translations": "node ./scripts/translations.js && git add './**/translations/**.json'",
```

Reply:
I thought on that, and is less harmful **but is doing the same**: impose a --bad-- workflow to the user; removes control if I don't wanna commit a translation file (personally, a very common scenario).

The way that this doesn't interfere is add the translation files that was modified **only** by the script; or been modified by the user and not staged for commit. But, the "problem" that this solves would happen once in a while, it would worth the effort?

As part of the CI of as script to been executed manually (or as personal/local hook, if your choice); I don't see other way 😕

Reply:
I updated to apply file changes to only added files.

Reply:
I see a commit that makes me understand that (a167fdd531d73d1ec43d9138bae53ae8e35ec33d), I had not review it but I don't see removing the `git add` command on the hook... so, I don't know how is that possible.

Trying to make it clear, I just tested with latest changes:
![capture](https://user-images.githubusercontent.com/5768813/47594888-08e40e80-d943-11e8-9970-70d1c332cfea.PNG)

Same behavior.